pandas/tests/groupby/test_groupby.py

```diff
@@ -2843,3 +2843,22 @@ def test_obj_with_exclusions_duplicate_columns():
     result = gb._obj_with_exclusions
     expected = df.take([0, 2, 3], axis=1)
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_numeric_only_std_no_result():
```

Comment:
Can you parameterize this test with `numeric_only` being True / False using `@pytest.mark.parametrize`.

Reply:
Done.